test/rules/grouped-imports/bad-order.ts.lint

```diff
@@ -0,0 +1,6 @@
+import {bar} from '../bar';
+
+import {foo} from 'foo';
+~~~~~~~~~~~~~~~~~~~~~~~~ [Import sources of different groups must be sorted by: libraries, parent directories, current directory]
```

Comment:
For now, I only add a single failure & fix in case of wrong orders. I've added a test with both cases though. More on this in a later comment.